lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -62,6 +73,21 @@ <h3 class="sr-only">${_("Print or share your certificate:")}</h3>
                             <span class="action-label">${_("Post on Facebook")}</span>
                         </a>
                       %endif
+                      % if share_settings.get('CERTIFICATE_TWITTER', False):
+                      <% twitter_url = 'https://twitter.com/intent/tweet?text=' + twitter_share_text + '&url='+ share_url%>
+                        <a
```

Comment:
Or, you can just move that value into your JS completely, since the reason we had it as a ``href`` was that a link would naturally use when JS wasn't available. On second thought, I'd recommend that.